manifest-build-tools/bintray.conf

```diff
@@ -0,0 +1,6 @@
+API="https://api.bintray.com"
+SUBJECT="rackhd-mirror"
+DISTRIBUTION="trusty"
```

Comment:
It's a bit confusing .

the hierarchy of bintray is SUBJECT --> REPO--> PACKAGE-->  FILE
so why REPO acts as  the script parameter ,  but FILE attribute(distribution/component/arch) lives inside a conf file ?

by the way, if user upload a generic file like static file (not debian package)?  how ?